tools/imageregistry/setup_imageregistry.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+kubectl create ns rukpak-system
```

Comment:
nit: this step might fail if this script is run on a cluster with rukpak already installed -- do we expect this script to always be run before any rukpak installation steps? would it make sense to check for an error here, and only conditionally create the namespace

Reply:
I see.  I’ll make ignore error, here.  Thanks,